soaks/Dockerfile.builder

```diff
@@ -1,11 +1,14 @@
 FROM docker.io/rust:1.58-bullseye@sha256:e4979d36d5d30838126ea5ef05eb59c4c25ede7f064985e676feb21402d0661b as builder
 RUN apt-get update && \
     apt-get dist-upgrade -y && \
-    apt-get -y install build-essential git clang cmake libclang-dev \
-    libsasl2-dev libstdc++-10-dev libssl-dev libxxhash-dev zlib1g-dev zlib1g
+    apt-get -y --no-install-recommends install build-essential git clang cmake libclang-dev \
```

Comment:
Same request here, with regard to recommends.

Reply:
This built all the way 👍